### PR TITLE
Fat-15178 Filter requests by pickup service point (vega) (TaaS)

### PR DESCRIPTION
**Pull request by victoriakor**
https://foliotest.testrail.io/index.php?/cases/view/15178
https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4457/allure/

### CODE REVIEW

File: cypress/e2e/requests/filter-request-by-pickup-service-point.cy.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import uuid from 'uuid';
+import testType from '../../support/dictionary/testTypes';
```
Comment (ernestyedigaryan1) [inline cypress/e2e/requests/filter-request-by-pickup-service-point.cy.js:2]: will be better to import this from '../../support/dictionary' like: 
`import { TestTypes, DevTeams } from '../../support/dictionary';`

File: cypress/support/fragments/requests/requests.js [hunk 2]
```diff
@@ -565,4 +565,22 @@ export default {
...
   }),
+
+  /* for miltiselect 'Pickup service point' we have to redefine attribute 'aria-labelledby' to make it unique,
+ because there are 4 elements with same 'aria-labelledby' on the page so the function 'createInteractor()'
+  in interactors\multi-select.js takes as argument the first one and it's not we needed
+  */
+  filterRequestsByServicePoints(servicePoint) {
+    // wait untill element is visible and interactable
+    cy.wait(3000);
+    cy.get('[id="req-pickup-service-point-filter"]').click();
```
Comment (ernestyedigaryan1) [inline cypress/support/fragments/requests/requests.js:576]: I think makes sense to make this element as a variable, because it is used twice here

**Review by dnovikau — CHANGES_REQUESTED** [review]
Please rebase this branch on top of master branch and push the changes as a single commit

File: cypress/e2e/requests/filter-request-by-pickup-service-point.cy.js (hunk 1 above)
Comment (victoriakor) [reply cypress/e2e/requests/filter-request-by-pickup-service-point.cy.js:2]: done)

**Comment by victoriakor** [comment]
Create a new one.

**Pull request closed without merging**